DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/ContractDetails/Index.cshtml

```diff
@@ -17,10 +17,7 @@
     <h1 class="nhsuk-heading-xl">@ViewData["Title"]</h1>
 
     <partial name="_ContractDetailsList" model="Model" />
-
-    <a class="nhsuk-button left-button-mobile-margin-bottom nhsuk-u-margin-right-2" asp-controller="Pricing" asp-action="Index" role="button">

```

Comment:
We should remove the pricing Controller too since we are no longer using it.

Reply:
Removed pricing Controller as well. Please verify.